pkg/analyze/analyzer.go

```diff
@@ -192,6 +200,9 @@ func Analyze(analyzer *troubleshootv1beta2.Analyze, getFile getCollectedFileCont
 		if err != nil {
 			return nil, err
 		}
+		for _, r := range results {
+			r.Strict = analyzer.DeploymentStatus.Strict.BoolVal
```

Comment:
This doesn't work
You need to use the index instead, see https://goplay.tools/snippet/h_Zmfb3Faz1
```suggestion
		for idx, _ := range results {
			range[idx].Strict = analyzer.DeploymentStatus.Strict.BoolVal
```